boards/arm/pinnacle_100_dvk/pinnacle_100_dvk.dts

```diff
@@ -113,14 +113,14 @@
 		compatible = "swir,hl7800";
 		status = "okay";
 		label = "hl7800";
-		mdm-reset-gpios = <&gpio1 15 0>;
-		mdm-wake-gpios = <&gpio1 13 0>;
-		mdm-pwr-on-gpios = <&gpio1 2 0>;
-		mdm-fast-shutd-gpios = <&gpio1 14 0>;
-		mdm-vgpio-gpios = <&gpio1 11 0>;
-		mdm-uart-dsr-gpios = <&gpio0 25 0>;
-		mdm-uart-cts-gpios = <&gpio0 15 0>;
-		mdm-gpio6-gpios = <&gpio1 12 0>;
+		mdm-reset-gpios = <&gpio1 15 (GPIO_OUTPUT | GPIO_OPEN_DRAIN)>;
+		mdm-wake-gpios = <&gpio1 13 (GPIO_OUTPUT | GPIO_OPEN_DRAIN)>;
```

Comment:
Also, what about `GPIO_ACTIVE_HIGH` or `GPIO_ACTIVE_LOW`

Reply:
You tell me, the driver itself wasn't setting GPIO_ACTIVE_HIGH or GPIO_ACTIVE_LOW so I just mimic what was in the pinconfig table into the dts.

Reply:
should be:
```
mdm-reset-gpios = <&gpio1 15 (GPIO_OPEN_DRAIN | GPIO_ACTIVE_LOW)>;
mdm-wake-gpios = <&gpio1 13 (GPIO_OPEN_SOURCE | GPIO_ACTIVE_HIGH)>;
mdm-pwr-on-gpios = <&gpio1 2 (GPIO_OPEN_DRAIN | GPIO_ACTIVE_LOW)>;
mdm-fast-shutd-gpios = <&gpio1 14 (GPIO_OPEN_DRAIN | GPIO_ACTIVE_LOW)>;
```

Reply:
Ok updated the dts and code.  Let me know if that all looks correct or not now.